airmozilla/popcorn/views.py

```diff
@@ -51,11 +42,72 @@ def event_meta_data(request):
 
     assert response.status_code == 302, response.status_code
 
+    return response.headers['location']
+
+
+def get_thumbnail(event):
+    image = event.picture and event.picture.file or event.placeholder_img
+    geometry = '160x90'
+    crop = 'center'
+
+    return thumbnail(image, geometry, crop=crop)
+
+
+@add_cors_header('*')
+@json_view
+def event_meta_data(request):
+    slug = request.GET.get('slug')
+    event = get_object_or_404(Event, slug=slug)
+    pattern = re.compile('(.+)\?')
+
+    base_url = get_video_url(event)
+    matches = pattern.match(base_url)
+    if matches:
+        video_url = matches.group()[:-1]
+    else:
+        video_url = base_url
```

Comment:
Don't use a regex. Use `urlparse.urlparse` and `urlparse.urljoin` to drop the randomish cache-buster thingy.